features/element-hiding.json

```diff
@@ -486,6 +486,10 @@
             {
                 "selector": ".clsy-c-advsection",
                 "type": "hide-empty"
+            },
+            {
```

Comment:
Should this not be domain specific (at least until we have another instance of it)?

Reply:
just made the change